qe/drivers.py

```diff
@@ -2190,10 +2137,10 @@ def __init__(
         self.d_two_e_integral = d_two_e_integral
         self.E0 = E0
         self.comm = comm
-        self.size = self.comm.Get_size()  # No. of processes running
+        self.world_size = self.comm.Get_size()  # No. of processes running
         self.rank = self.comm.Get_rank()  # Rank of current process
         # Dispatch wavefunction and save psi_local, dictionary of 2e integrals with local instance of class
-        if self.size == 1:  # One process running, use entire wavefunction
+        if self.world_size == 1:  # One process running, use entire wavefunction
             self.psi_i = psi_j
         else:  # Else, dispatch chunks of the wavefunction to all processes
             self.psi_i = dispatch_psi(comm, psi_j)
```

Comment:
I think we should always dispatch regardless of the number of processes.   The master thread should still do some work, and not only do the dispatch.
 But maybe we can discuss it, MPI expert may have more insight

Reply:
I think you're right as well. I also need to figure out where it makes the most sense to do the dispatching. Right now, I have it in this class by default. But it should probably be done when we load the wavefunction I'm thinking?